demo/vl-wizard.html

```diff
@@ -0,0 +1,193 @@
+                            <span is="vl-icon" data-vl-icon="arrow-left-fat" data-vl-before></span>
+                            Vorige
+                        </button>
+                        <button is="vl-button" type="button" slot="next-action">
+                            Volgende
+                        </button>
+                    </vl-wizard-pane>
+                    <vl-wizard-pane>
+                        <h3 is="vl-h3" slot="title">Stap 4</h3>
+                        <div is="vl-form-grid" is-stacked slot="content">
+                            <div is="vl-form-column" size="12">
+                                <label is="vl-form-label" for="disease" block>Reden van ziekte</label>
+                                <input id="disease" is='vl-input-field' block />
+                            </div>
+                        </div>
+                        <button is="vl-button-link" type="button" slot="previous-action">
+                            <span is="vl-icon" data-vl-icon="arrow-left-fat" data-vl-before></span>
+                            Vorige
+                        </button>
+                        <button is="vl-button" type="button" slot="next-action">
+                            Versturen
+                        </button>
+                    </vl-wizard-pane>
+                </div>
+            </vl-wizard>
+        </div>
+    </div>
+
+    <div>
+        <h2>Wizard met navigatie beperkingen</h2>
+        <pre>
+            <code class="line-numbers language-markup">
+            </code>
+        </pre>
+        <div class="demo">
+            <vl-wizard id="vl-wizard-checkbox">
+                <h2 is="vl-h2" slot="title">Wizard title</h2>
+                <p slot="header">You're a wizard Harry</p>
+                <div slot="panes">
+                    <vl-wizard-pane id="vl-wizard-checkbox-pane-1" data-vl-next-pane-disabled>
+                        <h3 is="vl-h3" slot="title">Stap 1</h3>
+                        <div is="vl-form-grid" is-stacked slot="content">
+                            <div is="vl-form-column" size="12">
+                                <vl-checkbox id="vl-wizard-checkbox-pane-1-next" label="Volgende toegelaten"></vl-checkbox>
+                            </div>
+                        </div>
+                        <button is="vl-button" type="button" slot="next-action">
+                            Volgende
+                        </button>
+                    </vl-wizard-pane>
+                    <vl-wizard-pane id="vl-wizard-checkbox-pane-2" data-vl-previous-pane-disabled data-vl-next-pane-disabled>
+                        <h3 is="vl-h3" slot="title">Stap 2</h3>
+                        <div is="vl-form-grid" is-stacked slot="content">
+                            <div is="vl-form-column" size="12">
+                                <vl-checkbox id="vl-wizard-checkbox-pane-2-previous" label="Vorige toegelaten"></vl-checkbox>
```

Comment:
ik zou denken dat het not done is een wizard om niet toe te laten om terug te gaan ...

Reply:
Dan gebruik je de feature niet? Stel dat er in de vorige stap een actie gebeurt is die niet zomaar terug gedraaid kan/mag worden en/of opnieuw kan uitgevoerd worden. Dan kan de vorige stap afgezet worden. Het is een feature die niet gebruikt moet worden.

Reply:
een stap zelf moet gewoon zeggen "ik ben disabled" en "ik ben compleet", al de rest moet volgens mij in wizard zitten